lookout/style/format/features.py

```diff
@@ -232,9 +243,12 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
             try:
                 vnodes, parents = self._parse_file(contents, uast, file.path)
             except AssertionError as e:
-                self._log.warning("could not parse file %s with error \"%s\', skipping",
+                self._log.warning("could not parse file %s with error '%s', skipping",
                                   file.path, e)
-                continue
+                if self.debug_parsing:
+                    input("press Enter to continue")
```

Comment:
How about `import ipdb; ipdb.set_trace()` then?

Reply:
I went for printing the trace directly instead, it proved to be the fatest during debugging. I can change if needed.